### ISSUE TITLE
[KEYBOARD]: Text Appointments - Skip link targets need tabindex -1 for IE11 to behave properly

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2429]
## Description

Our On this page links are designed to let users skip to specific content sections. IE11 does not focus the content container as expected, and our skip links should be considered broken for IE11 users who prefer to navigate by keyboard. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As an IE11 keyboard user, I want to press `ENTER` on any of three links, and be able to press `TAB` again and have focus settle on the correct content.
* As a non IE user, I do not want my visual or aural experience to be broken or changed.

## Environment

* Windows 10
* IE11
* staging.va.gov profile page 

## Possible Fixes (optional)

Adding a `tabindex="-1"` to the target container seems to fix the issue. I am not concerned if we use a script or hard code this value as long as it does not interfere with user experience in non-IE11 browsers.

```html
<h2 class="va-profile-heading" tabindex="-1">Contact information</h2>
```

**Prior Art**
* https://github.com/uswds/uswds-site/pull/230
* https://github.com/Automattic/_s/blob/master/js/skip-link-focus-fix.js

## WCAG or Vendor Guidance (optional)

* [Understanding Success Criterion 2.4.1 (W3C)
](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-14 at 3 16 22 PM](https://user-images.githubusercontent.com/934879/66780256-ffc2b900-ee95-11e9-9044-3d7a8b0ad06c.png)

**Comment by lkoenigsberg** [comment]
This will move to the Authenticated Experience board for consideration as we merge account and profile to a single page and redesign. (cc/ @Justin-Pickett  )
